back-end/requests/put/add_to_ban.js

```diff
@@ -0,0 +1,56 @@
+let Group = require("../../models/groups.model");
+
+const add_to_ban = async (req, res,next) => {
+  const group_id = req.params.group_id
+  const user_id = req.params.user_id
+
+  let members
+  let banned_members
```

Comment:
should declare `owners` here as well. It seems to work fine like this but I believe assigning a value to `owners` without declaring it makes it a global variable of some sort.